bionic/executor.py

```diff
@@ -44,6 +44,31 @@ def submit(self, task_config, fn, *args, **kwargs):
         ).submit()
 
 
+class FakeAipExecutor:
```

Comment:
This PR adds fake code across three different non-test files. Could we just put all this fake AIP stuff into one file? Possibly in the test directory?

Also, do we actually need a `FakeTask` class, or can we just inline the code from `submit`?

Reply:
I'll move all the fake code in a fakes test file. 

I thought about `FakeTask` class more. I think it would be nice to keep `FakeTask` that inherits `Task` so that we can use the `Task._stage` method.